app/models/role.rb

```diff
@@ -171,6 +175,7 @@ def clone(role_params = {})
   end
 
   def locked?
+    return false if self.class.bypass_locking
```

Comment:
I agree, if we're disabling locking then no roles are locked and it makes sense to return false.